src/fidasim.f90

```diff
@@ -1970,6 +1983,10 @@ subroutine read_inputs
     nlambda_wght=0
     lambdamin_wght=0
     lambdamax_wght=0
+    adaptive=0
+    p_adaptive=0
+    max_cell_splits=1
```

Comment:
is `1` a good default here?

Reply:
1 should be the default and the minimum number of cell splits since it would correspond to no splitting